Backend/Controllers/InventoryController.cs

```diff
@@ -62,4 +62,21 @@ public ActionResult DeleteItem(uint id)
         return Ok($"Deleted item with ID {id}.");
     }
 
+    [HttpGet("test")]
+    public ActionResult TestConnection()
+    {
+        return Ok("Connection to InventoryController successful.");
+    }
+
+    [HttpPut]
+    [Route("update-sales/{saleAmount}")]
+    public ActionResult UpdateSales([FromBody] Item updateItemSales, uint saleAmount)
```

Comment:
I'm not opposed to this endpoint existing, but we do already have the generic `Update` endpoint and `ItemService.Update(Item)`.

My concern is if we do this approach it would intuitively mean that we end up doing this for every field, and that would get messy I think.

Could you consider testing updating the sales field of an item with the normal `update` endpoint?  If that works, then I think we should take that approach.

That approach would be don't care what field has changed, let the front end do whatever it wants to the item, and update that item.